gp-includes/things/translation.php

```diff
@@ -31,6 +31,14 @@ class GP_Translation extends GP_Thing {
 	public $translation_4;
 	public $translation_5;
 	public $user_id;
+	/**
+	 * User id of user (validator) who last changed the status of the translation.
+	 *
+	 * @since 2.1.0
+	 *
+	 * @var $user_id_last_modified
```

Comment:
Should be `@var int`, see https://phpdoc.org/docs/latest/references/phpdoc/tags/var.html.